fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -2411,6 +2417,10 @@ struct DDQueueImpl {
 						    .detail("HighestPriority", highestPriorityRelocation)
 						    .detail("BytesWritten", self->moveBytesRate.getTotal())
 						    .detail("BytesWrittenAverageRate", self->moveBytesRate.getAverage())
+						    .detail("RelocateToSourceRatio",
+						            self->relocateTotalCount
+						                ? 1.0 * self->relocateToSourceCount / self->relocateTotalCount
+						                : 0)
```

Comment:
To clarify, do `relocateTotalCount` and `relocateToSourceCount` track the numbers over the lifetime of a DD task, i.e., they're not reset periodically -- they each start at 0 when a DD task (re)starts and keep counting up?

I think it might be more useful to print the two counts since that gives us more information and a ratio can always be calculated in a dashboard if desired out of those numbers; but if we print a ratio, then we can't interpret the actual numbers from that.

Reply:
Yes, they don't reset periodically, that's why I chose `int64_t` as type.
Okay. I split into `RelocateToSourceRatio` into `RelocatorTotalCount` and `RelocatorToSourceTeamCount`